crates/saya/core/src/prover/stone_image.rs

```diff
@@ -50,8 +51,8 @@ impl StoneProver {
         static STONE_PROVER: OnceCell<(anyhow::Result<String>, anyhow::Result<String>)> =
             OnceCell::const_new();
 
-        let source = "neotheprogramist/state-diff-commitment";
-        let verifier = "neotheprogramist/verifier:latest";
+        let source = "piniom/state-diff-commitment";
+        let verifier = "piniom/verifier:latest";
```

Comment:
@neotheprogramist we should update this later once `dojo-os` will be ready on your side with latest changes. 👍